web_ui/packages/smart-tools/src/utils/opencv-loader.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (C) 2022-2025 Intel Corporation
+// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
+
+export default async (): Promise<typeof import('../opencv/4.9.0/opencv.js')> => {
```

Comment:
update opencv loader to use ESM so it doesnt necessarily be used by workers as it was before